--- libgit2-0.23.3/CMakeLists.txt.orig	2015-10-18 14:13:01.342360169 +0200
+++ libgit2-0.23.3/CMakeLists.txt	2015-10-18 14:15:26.742357088 +0200
@@ -588,7 +588,11 @@
 
 	ENABLE_TESTING()
 	IF (WINHTTP OR OPENSSL_FOUND OR SECURITY_FOUND)
+		IF (ONLINE_TESTS)
 		ADD_TEST(libgit2_clar libgit2_clar -ionline)
+		ELSE ()
+			ADD_TEST(libgit2_clar libgit2_clar -xonline -xnetwork::remote::remotes)
+		ENDIF ()
 	ELSE ()
 		ADD_TEST(libgit2_clar libgit2_clar -v)
 	ENDIF ()
